### ISSUE TITLE
9.5 Support

### ISSUE THREAD

**Issue by 173210** [roxas75/rxTools#333]
Here is an ARM11 kernel exploit.
https://github.com/JustPingo/KernelTimeMachine/blob/master/source/libsu.c

**Comment by AlbertoSONIC** [comment]
We won't support 9.5 with the arm11 kernel exploit as we still need an arm9 exploit. If people want to use rxTools, then they can downgrade to 9.2 and use it ;)

**Comment by 173210** [comment]
@AlbertoSONIC The vulnerability is available until 9.5, you see?

**Comment by 173210** [comment]
I mean, firmlaunchax.

**Comment by AlbertoSONIC** [comment]
Still there's no reason to support 9.5 as if an user is on 10.x, why would him dg to 9.5 instead of 9.2?

**Comment by 173210** [comment]
@AlbertoSONIC Don't you love the newer version and it's super easy to implement 9.5 support because rxTools doesn't depend on 9.2 thanks to the wide-range support (4.5-9.2?)

**Comment by 173210** [comment]
I think it's reasonable to add 9.5 to the supported firmware list.

**Comment by AlbertoSONIC** [comment]
Mmmm I'm not sure... This is a minor thing... Do you know of the arm9loaderhax? That would extend the compatibility to 10.3, which would mean the end of the emunand! I think that this is way more important than adding support to 9.5... 

Also, AFAIK none is on 9.5 (there was no reason in keeping an old unhackable firmware) ;)

**Comment by 173210** [comment]
@AlbertoSONIC Yes, it's a minor thing, and easy to implement. At first, you should start from the easiest part.

**Comment by AlbertoSONIC** [comment]
Ok if you think it's worth it, than go ahead... But I suggest you to wait for that memchunkhax2 to become stabler ;)

**Comment by 173210** [comment]
@AlbertoSONIC Who makes it stabler? I think this should be ropened.

**Comment by AlbertoSONIC** [comment]
A friend of mine ;)

**Comment by ApacheThunder** [comment]
9.4 support would be great. But now that I think about it, it would lock me out of some older CFW/homebrew that use the older memchunkhax exploit. It would be interesting. Not something I'd attempt until I get my NAND mod fixed. :P
